examples/kafka_spark_demo/feature_repo/streaming_features_tutorial.ipynb

```diff
@@ -0,0 +1,877 @@
+    "\n",
+    "Let's also inspect the historical features for the stream feature view."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   driver_id                  event_timestamp  conv_percentage  acc_percentage\n",
+      "0       1001        2021-04-12 10:59:42+00:00        52.114902       75.165855\n",
+      "1       1003        2021-04-12 16:40:26+00:00        18.885477       34.473606\n",
+      "2       1004        2021-04-12 15:01:12+00:00        29.649216       93.530525\n",
+      "3       1002        2021-04-12 08:12:10+00:00         8.901370       21.263689\n",
+      "4       1001 2022-06-17 11:32:55.580852+00:00        40.458847       40.757076\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/felixwang/feast/sdk/python/feast/stream_feature_view.py:98: RuntimeWarning: Stream Feature Views are experimental features in alpha development. Some functionality may still be unstable so functionality can change in the future.\n",
+      "  warnings.warn(\n"
+     ]
+    }
+   ],
+   "source": [
+    "entity_df = pd.DataFrame.from_dict(\n",
+    "    {\n",
+    "        \"driver_id\": [1001, 1002, 1003, 1004, 1001],\n",
+    "        \"event_timestamp\": [\n",
+    "            datetime(2021, 4, 12, 10, 59, 42),\n",
+    "            datetime(2021, 4, 12, 8, 12, 10),\n",
+    "            datetime(2021, 4, 12, 16, 40, 26),\n",
+    "            datetime(2021, 4, 12, 15, 1, 12),\n",
+    "            datetime.now()\n",
+    "        ]\n",
+    "    }\n",
+    ")\n",
+    "training_df = store.get_historical_features(\n",
+    "    entity_df=entity_df,\n",
+    "    features=[\n",
+    "        \"driver_hourly_stats_stream:conv_percentage\",\n",
+    "        \"driver_hourly_stats_stream:acc_percentage\",\n",
+    "    ],\n",
+    ").to_df()\n",
+    "print(training_df.head())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is exactly what we need to train our model!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Ingesting and transforming data from a Kafka topic\n",
+    "\n",
+    "Now let's switch to the perspective of an ML engineer. We see that a data scientist has just a stream feature view, and we are now responsible for ensuring that features from the stream are available in production.\n",
+    "\n",
+    "We will use Spark Structured Streaming to ingest from a Kafka topic, transform the data, and then push the data to the online store.\n",
```

Comment:
The fact that we're using Spark should be called out in the definition of the stream feature view, because that's the mode we're using (it's not like we could use anything else if we use mode="spark)

Reply:
done